images/common/install-sambacc-common.sh

```diff
@@ -31,26 +31,48 @@ install_sambacc() {
 
 
     local action=install-from-copr-repo
-    if [ "${#wheels[@]}" -gt 1 ]; then
-        echo "more than one wheel file found"
-        exit 1
-    elif [ "${#wheels[@]}" -eq 1 ]; then
-        action=install-wheel
+    local source_found=0
```

Comment:
Why not just leave `action` unset and then do a test like `[ -z $action ]` for the other if block? No need for a 2nd variable.

Reply:
good point! done.